src/lobby/react.tsx

```diff
@@ -8,20 +8,64 @@
 
 import React from 'react';
 import Cookies from 'react-cookies';
-import PropTypes from 'prop-types';
 import { Client } from '../client/react';
 import { MCTSBot } from '../ai/mcts-bot';
 import { Local } from '../client/transport/local';
 import { SocketIO } from '../client/transport/socketio';
-import { LobbyConnection } from './connection';
+import { GameComponent, LobbyConnection } from './connection';
 import LobbyLoginForm from './login-form';
-import LobbyMatchInstance from './match-instance';
+import LobbyMatchInstance, { MatchOpts } from './match-instance';
 import LobbyCreateMatchForm from './create-match-form';
+import { LobbyAPI } from '../types';
 
-const LobbyPhases = {
-  ENTER: 'enter',
-  PLAY: 'play',
-  LIST: 'list',
+enum LobbyPhases {
+  ENTER = 'enter',
+  PLAY = 'play',
+  LIST = 'list',
+}
+
+type RunningMatch = {
+  app: ReturnType<typeof Client>;
+  matchID: string;
+  playerID: string;
+  credentials?: string;
+};
+
+type LobbyProps = {
+  gameComponents: GameComponent[];
+  lobbyServer?: string;
+  gameServer?: string;
+  debug?: boolean;
```

Comment:
This should probably align with the `debug` option that can be passed to the client functions:

https://github.com/boardgameio/boardgame.io/blob/af435611685d72e3ea6e1a81909c9bb6522a18dd/src/client/client.ts#L104

This allows users to pass the full debug panel implementation if they want to include it in production builds ([docs](https://boardgame.io/documentation/#/debugging?id=using-the-debug-panel-in-production)).

Reply:
Done in a994a43.